src/test/kotlin/org/neo4j/graphql/AugmentationTest.kt

```diff
@@ -149,13 +151,23 @@ class AugmentationTest {
     @Test
     fun testMutationForRelations() {
         val ctx = Translator.Context(mutation = Translator.CRUDConfig(enabled = true, exclude = listOf("Person0","Person1","Person2","Person3", "Person4")), query = Translator.CRUDConfig(enabled = false))
-        augmentedSchema(ctx, typeFor("Person5")).let {
+        augmentedSchema(ctx, typeFor("Person5"), typeFor("Movie")).let {
+            assertNotNull(it)
             assertEquals("",it.ordering)
             assertEquals("",it.filterType)
-            assertEquals("createPerson5(id:ID!) : Person5 ",it.create)
-            assertEquals("mergePerson5(id:ID!) : Person5 ",it.merge)
-            assertEquals("updatePerson5(id:ID!) : Person5 ",it.update)
-            assertEquals("deletePerson5(id:ID!) : Person5 ",it.delete)
+            assertEquals("addPerson5Movies(id:ID!, movies:[ID]!) : Person5", it.create)
```

Comment:
I think it has to be [ID!] i.e. allow empty lists (which wouldn't do anything) but not a list of null

Reply:
I used the same convention as explained here:
https://github.com/neo4j-graphql/neo4j-graphql#auto-generated-mutations
Otherwise I think it should be [ID!]! (https://graphql.org/learn/schema/)

Reply:
Yep the latter is probably correct.

Reply:
ok